src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -97,6 +97,27 @@ public function prepare_items() {
 
 		$args['posts_per_page'] = $per_page;
 
+		if ( 'scheduled' === $this->tab->get_slug() && isset( $_REQUEST['s'] ) && ! empty( $_REQUEST['s'] ) ) {
+			$args['meta_query'] = array(
+				'relation' => 'OR',
+				array(
+					'key' => '_tribe_aggregator_source_name',
+					'value' => $_REQUEST['s'],
+					'compare' => 'LIKE'
+				),
+				array(
+					'key' => '_tribe_aggregator_import_name',
```

Comment:
:warning: Array double arrow not aligned correctly; expected 5 space(s) between "'key'" and double arrow, but found 1.

<sub>hash: deca2851901a22e75005a9dbf1a715eb</sub>